pkgs/development/libraries/qt-5/modules/qtwebengine.nix

```diff
@@ -101,6 +101,16 @@ EOF
 
     substituteInPlace src/3rdparty/chromium/sandbox/mac/BUILD.gn \
       --replace 'libs = [ "sandbox" ]' 'libs = [ "/usr/lib/libsandbox.1.dylib" ]'
+
+    # patch for CVE-2019-13720, can be removed when it is included in the next upstream release
+    # https://bugreports.qt.io/browse/QTBUG-1019226
+    cd src/3rdparty
+    patch -p1 < ${fetchpatch {
```

Comment:
I think what you are looking here is the `extraPrefix` argument to fetchpatch:
https://github.com/NixOS/nixpkgs/blob/cb8423d19ba6ab9aa06e5c9fcf0fbae61aa9a12b/pkgs/build-support/fetchpatch/default.nix#L12

The patch might fail for Qt 5.9 or 5.11, so we shouldn't try to apply the same one in every Qt version.

Reply:
Thanks, of course that's the better solution.